packages/storybook/stories/va-button.stories.jsx

```diff
@@ -51,34 +51,40 @@ const Template = ({
   );
 };
 
-export const Primary = Template.bind({});
+export const Primary = Template.bind(null);
```

Comment:
These changes were made to have the Story tab display the stories correctly in Canvas.